src/ngraph/runtime/cpu/op/conv_relu.hpp

```diff
@@ -45,15 +54,39 @@ namespace ngraph
             CoordinateDiff m_padding_below;
             CoordinateDiff m_padding_above;
             Strides m_data_dilation_strides;
+        };
 
-        private:
-            ConvolutionRelu(const std::shared_ptr<Node>& data_batch,
-                            const std::shared_ptr<Node>& filters,
-                            const Strides& window_movement_strides,
-                            const Strides& window_dilation_strides,
-                            const CoordinateDiff& padding_below,
-                            const CoordinateDiff& padding_above,
-                            const Strides& data_dilation_strides);
+        /// \brief Relu(Convolution) forward prop for batched convolution operation with bias
+        class ConvolutionBiasRelu : public util::RequiresTensorViewArgs
+        {
+        public:
+            ConvolutionBiasRelu(const std::shared_ptr<op::ConvolutionBias>& conv);
+
+            ConvolutionBiasRelu(const std::shared_ptr<Node>& data_batch,
+                                const std::shared_ptr<Node>& filters,
+                                const std::shared_ptr<Node>& bias,
+                                const Strides& window_movement_strides,
+                                const Strides& window_dilation_strides,
+                                const CoordinateDiff& padding_below,
+                                const CoordinateDiff& padding_above,
+                                const Strides& data_dilation_strides);
+
+            const Strides& get_window_movement_strides() const { return m_window_movement_strides; }
+            const Strides& get_window_dilation_strides() const { return m_window_dilation_strides; }
```

Comment:
do we even need dilation if we ignore any convolution whose dilation strides aren't equal to 1?